index.d.ts

```diff
@@ -93,7 +110,7 @@ interface AgentConfigOptions {
   logLevel?: LogLevel;
   logger?: Logger;
   metricsInterval?: string | number; // TODO: Do we officially want to support numbers?
-  payloadLogFile?: string; // TODO: Do we want to advertise this?
+  payloadLogFile?: string; // TODO: Should we advertise this API?
```

Comment:
Any reason _not_ to? It's a helpful debugging feature. 🤔

Reply:
Good point. I was just thinking that it wasn't in our official docs, but now that I just checked I can see that it's in the README.md file.